extension/src/Runner/testModel.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { Range } from "vscode";
+import { TestLevel, TestResult } from "../protocols";
+
+export interface ITestInfo {

```

Comment:
The members are part of TestSuite. Can we make that an interface that extends this one?